### PR TITLE
roles document updated

### PR DESCRIPTION
**Pull request by khanzadyan**

### CODE REVIEW

File: content/_guides/managing-user-roles-in-a-tenant.md [hunk 1]
```diff
@@ -1,26 +1,19 @@
...
-This document explains [what roles and permissions are](#roles-and-permissions)
-and how [to set roles in the UI](#setting-user-roles).
+{{page.description}}
 
 ## Roles and permissions
 
-A tenant admin can
-control user rights in his tenant by configuring user roles. A role is a user
-attribute that defines a set of permissions for the user. Contract roles define
+A tenant admin can control user rights in his tenant by configuring user roles.
```
Comment (HannaTrotsenko) [inline content/_guides/managing-user-roles-in-a-tenant.md:15]: his/her?

File: content/_guides/managing-user-roles-in-a-tenant.md [hunk 2]
```diff
@@ -33,8 +26,8 @@ The default **contract roles** are:
 The default **workspace roles** are:
 
 *   **Owner** - Users with this role can edit the workspace, edit [flows](/getting-started/integration-flow), start/stop flows, change flows from/to [real-time/ordinary](realtime-flows) status, and edit credentials in a workspace.
```
Comment (HannaTrotsenko) [inline content/_guides/managing-user-roles-in-a-tenant.md:28]: >  edit the workspace
maybe it makes sense to clarify what exactly means `edit workspace` - rename, add/invite new members, manage members roles

File: content/_guides/managing-user-roles-in-a-tenant.md [hunk 3]
```diff
@@ -57,8 +50,8 @@ drop-down menu.
 As a workspace Owner and Admin you can set the user role while inviting them to
 the Contract and Workspace.
 
-> **Please Note** You can invite to Contract and Workspace only if your have
-> **Owner** or **Admin** role in the contract.
+> **Please Note** You can invite to Contract and Workspace you have either **Owner**
+> or **Admin** role in the contract.
```
Comment (HannaTrotsenko) [inline content/_guides/managing-user-roles-in-a-tenant.md:54]: Contract Admin doesn't have permissions to add/invite new members to the Contract

Comment (HannaTrotsenko) [reply content/_guides/managing-user-roles-in-a-tenant.md:54]: The same in this sections

> As a contract **Owner** or **Admin** you can modify roles of any user in the
contract by visiting **Members** section of the **Contract Settings** and
clicking to edit (the pencil).

Comment (HannaTrotsenko) [reply content/_guides/managing-user-roles-in-a-tenant.md:54]: `{
                    "i18n": {
                        "en": "Admin"
                    },
                    "role": "admin",
                    "permissions": [
                        "contracts.workspace.create",
                        "contracts.workspace.listAll",
                        "contracts.workspace.delete",
                        "contracts.repository.edit",
                        "contracts.devTeam.edit",
                        "global.auth_clients.create",
                        "global.auth_clients.get",
                        "global.auth_clients.edit",
                        "global.auth_clients.delete"
                    ],
                    "scope": "contracts"
                },`

Comment (HannaTrotsenko) [reply content/_guides/managing-user-roles-in-a-tenant.md:54]: And remove pls Admin from

> As a contract **Owner** or **Admin** you can modify roles of any user in the
contract by visiting **Members** section of the **Contract Settings** and
clicking to edit (the pencil).

**Comment by khanzadyan** [comment]
Thanks @HannaTrotsenko for your comments. I will get to it.

File: content/_guides/managing-user-roles-in-a-tenant.md (hunk 1 above)
Comment (khanzadyan) [reply content/_guides/managing-user-roles-in-a-tenant.md:15]: I think I would prefer to write not his or her but "tenant admin can control user rights in a tenant"

**Pull request closed without merging**